product_docs/docs/pgd/5/consistency/column-level-conflicts/04_timestamps.mdx

```diff
@@ -0,0 +1,103 @@
+---
+navTitle: Timestamps
+title: Timestamps
+redirects:
+  - /pgd/latest/bdr/column-level-conflicts/
+---
+
+## Current versus commit timestamp
+
+An important decision is which timestamp to assign to modified columns.
+
+!!! Note
```

Comment:
Too Many Notes. The page looks like it wants a ransom paid. :)

Reply:
Any suggestions? This was my attempt to integrate the timestamp notes into the text. I am not sure how I would integrate these notes into the text otherwise.

Reply:
Ask "is it a note or just an oddly marked out paragraph"....

There's a note on line 29 which talks about future features which can probably be de-noted and then made into a non-future referencing statement.
Line 22 is just important information, not an extra note.

And now we have less notes :)

Reply:
I am using the doc you wrote for this. I finally think I understand what is going on here with timestamps and column-level conflict resolution after reading it.